public/locales/en.json

```diff
@@ -14,7 +14,15 @@
     "forgotPassword": "Forgot Password ?",
     "enterEmail": "Enter Email",
     "enterPassword": "Enter Password",
-    "doNotOwnAnAccount": "Do not own an account?"
+    "toastError1": "Talawa-API service is unavailable. Is it running? Check your network connectivity too.",
+    "toastError2": "Captcha Error!",
+    "toastError3": "Please, check the captcha.",
+    "toastError4": "Something went wrong, Please try after sometime.",
+    "toastError5": "Password and Confirm password mismatches.",
+    "toastError6": "Fill all the Details Correctly.",
+    "toastError7": "Sorry! you are not Authorised!",
+    "toastError8": "User not found!",
```

Comment:
Why is it toastError1..8 and not a more appropriate name based on the string it represents.